tools/chpl-language-server/src/chpl-language-server.py

```diff
@@ -355,6 +356,44 @@ def unregister_workspace(self, uri: str):
         if uri in self.configurations:
             del self.configurations[uri]
 
+    def _get_enum_inlays(self, decl: NodeAndRange) -> List[InlayHint]:
+        if not self.enum_inlays:
+            return []
+
+        node = decl.node
+        if not isinstance(node, chapel.EnumElement):
+            return []
+
+        enum_value = node.assigned_value()
+        inlay_value: str | None = None
+        if isinstance(enum_value, chapel.IntParam):
+            inlay_value = str(enum_value.value())
+        elif isinstance(enum_value, chapel.UintParam):
+            inlay_value = str(enum_value.value())
+
+        if not inlay_value:
+            return []
+
+        node_init = node.init_expression()
+        edits = []
+        if isinstance(node_init, chapel.Literal):
+            return []
+        elif isinstance(node_init, chapel.AstNode):
+            position = location_to_range(node_init.location()).end
+            value = " /* " + inlay_value + " */"
+        else:
+            position = location_to_range(node.location()).end
+            value = " = " + inlay_value
+            edits = [TextEdit(Range(position, position), value)]
```

Comment:
what happens if a user has `v /*= 1*/`, should we suppress the inlay in that case?

Note that is what we do with the end block markers today

Reply:
I chose not to do this because there are many ways you can write this.

1. Do you silence exactly when `/*= 1*/` appears?
   
   **counterexample**: `/* computes to 1 */` or just `/* 1 */`
2. Do you silence when any comment is present?
   
   **counterexample**: `red /* my favorite color! */`
3. Do you silence when a comment is present, and the value is included in it?

   **counterexample**: `red /* 1 out of 3 primary RGB colors */`

My intuition is that this pattern doesn't occur often(why would you annotate enum constants' values like this if you can just assign them?) and that it's best not to get too clever to avoid strange behavior.